packages/core/types/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@parcel/cache": "2.7.0",
+    "@parcel/core": "2.7.0",
```

Comment:
`@parcel/types` doesn't depend on `@parcel/core`, and doing so would cause a circular dependency to be introduced. I guess maybe it should be a peer dependency, but even then it's a little weird...